src/c4v/scraper/spiders/el_pitazo.py

```diff
@@ -0,0 +1,73 @@
+# Internal imports
+import c4v.scraper.utils as utils
+
+# External imports
+import scrapy
+
+# Python imports
+from typing import List, Dict, Any
+
+
+class ElPitazoSpider(scrapy.Spider):
+    """
+        Spider to scrape ElPitazo data 
+    """
+
+    name = "el_pitazo"
+
+    start_urls = []
+
+    def parse(self, response) -> Dict[str, Any]:
+        """
+            Returns a dict like structure with the following 
+            fields:
+                + title
+                + date
+                + categories
+                + body
+                + author 
+                + tags
+        """
+
+        # These are simple properties, just get its text with a valid
+        # selector
+        title = response.css(".tdb-title-text ::text").get() or ""
+        date =  response.css(".entry-date ::text").get() or ""
+        author = response.css(".tdb-author-name ::text").get() or ""
+
+        body = self._get_body(response)
+
+        tags = self._get_tags(response)
+
+        # categories
+        categories = response.css(".tdb-entry-category")
```

Comment:
nit: this line can simply be
`categories = response.css(".tdb-entry-category ::text").getall()`
this returns a list with the text found on each element of class "tdb-entry-category"